guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -39,8 +39,11 @@ endif::[]
 ifdef::satellite[]
 [NOTE]
 ====
-{Project} supports DEFAULT and FIPS crypto-policies.
+* {Project} supports DEFAULT and FIPS crypto-policies.
 The FUTURE crypto-policy is not supported for {Project} and {SmartProxy} installations.
+* In-place upgrade of {EL} systems in FIPS mode is not supported by {Team}.
+Turning FIPS off, upgrading from {EL} 7 to {EL} 8, and then turning FIPS on is not supported either.
+Instead, install {Project} {ProductVersion} on a freshly installed {EL} 8 system with FIPS mode enabled.
```

Comment:
Following this, you'd end up with a new, empty system. Which is probably not what people want :)

Can we link users up to the next section ("5. Migrating Satellite to a New Red Hat Enterprise Linux System") somehow? Something like "Follow the 'Migrating to new system' procedure with the new system in FIPS mode"?